v2/array/array.go

```diff
@@ -15,6 +15,10 @@ func countMap(arrs ...[]string) (result map[string]int) {
 	return
 }
 
+func MergeString(arrs ...string) (result []string) {
```

Comment:
I would not call it merging, more like concatenating, but it does creating a list of items. Why not call it `StringArray("a", "b", "c")`?

Reply:
Unused aswell